types/inquirer/inquirer-tests.ts

```diff
@@ -1,765 +1,49 @@
-import chalkPipe = require("chalk-pipe");
-import * as inquirer from "inquirer";
-import { from as rxjsFrom, Observable, Observer } from "rxjs";
-
-/**
- * Bottom Bar example
```

Comment:
I had copied all of these examples directly from https://github.com/SBoudrias/Inquirer.js/tree/master/packages/inquirer/examples and put each one inside their own `{}` scope, so as not to affect the other test cases. I'm not sure where you've gotten these new tests, but I think it makes more sense if they came from the provided examples in their GitHub repo than hand-typed. If you want to add more custom tests at the very bottom, that's fine, but why were all of these original tests removed?

I'm shocked, actually, that you've effectively just added a bunch more definitions, but deleted 99% of the tests in the process. Why was this done? I'm extremely concerned about this.

Reply:
The code-examples provided here are very verbose.
The test-file is intended for testing each use-case of a module (in this case prompting questions, creating a prompt-module, creating a custom UI and stuff).

To me it's ways more clear to have these things covered with some brief test-snippets rather than using a bunch of code provided by a package's documentation.

If you want me to add the examples back I can do so, though.

Reply:
I'm not sure these DT tests can be compared to traditional app tests. The whole point of tests is to increase confidence. If the author of a library says it should work a certain way and provides example usage, it stands to reason that these examples should work in TypeScript w/o any kind of errors. The only way to increase my confidence to 100% is to include these examples in the DT tests; otherwise, I wonder if something was missed.

Are you 100% sure all of these examples work w/o errors? If so, how are you so sure? You already bumped the major version of these DT types because you were worried that existing implementations would break, so that tells me you're not 100% confident, defeating the whole purpose of the tests in the first place.

If you really want to delete all of these tests, then I suggest this PR be squashed down into 2 commits. The first commit would be all of your changes with the pre-existing tests preserved and this first commit would get a green checkmark on the PR before pushing the 2nd commit, where most of those tests are deleted. This would give me 100% confidence that this PR didn't break anything, but it would make future changes destroy that confidence (not my recommendation).